lib/prosperworks/person.rb

```diff
@@ -31,5 +31,10 @@ def self.find_by_email(email)
       handle_response(new, response)
     end
 
+    def opportunities
+      uri = self.class.get_uri(self.class.api_name, id, 'related/opportunities')
+      response = self.class.send_request('get', uri)
+      Opportunity.handle_multiple_response(response)
```

Comment:
Thinking about this, can we store the value locally, so that each call to this function doesn't result in another API call? So something like:
```
def opportunities(force_refresh=false)
  return @opportunities if !@opportunities.nil? && force_refresh == true

  uri =  self.class.get_uri(self.class.api_name, id, 'related/opportunities')
  ...
  @opportunities = Opportunity.handle_multiple_response(response)
end
```